gauntlet/gauntlet.go

```diff
@@ -16,6 +16,10 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+var (
```

Comment:
@RodrigoAD we should find a better location for the Gauntlet Go wrapper, any ideas? 

Maybe https://github.com/smartcontractkit/gauntlet-core